app.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env sh
```

Comment:
we wont need app.sh file
as in the tasks we have to defined execute command, so this could directly go there.

so my vision for this repo was with multiple scripts.

Reply:
I ll remove it! Thanks @harshad16